docs/_future/listening_responding_functions.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Listening & responding to functions
```

Comment:
I'm wondering if we can tie this with the "Manifest functions" section in some way. Currently they seem a bit segmented and I can imagine that might feel confusing to newcomers that are trying to understand the full end-to-end flow of 1) initializing a function in the Manifest and then 2) creating some sort of listener or handler for the function that can then execute custom functionality with the desired inputs and outputs defined in the initialization.

Reply:
I would prefer to keep them separate since the manifest information will be exactly the same for the JAVA sdk, we will potentially move this out of the python doc to a centralized location